### ISSUE TITLE
Questions about the PSNR result of the project

### ISSUE THREAD

**Issue by BeamReflection** [InterDigitalInc/CompressAI#318]
I want to know how can I get the result like the picture in the project. I notice that some of them can exceed 40dB, but i can just only get a 30dB result. Of course it will increase the bpp, but that's fine. 
Just increase/decrease the value of LAMBDA is OK? It seems that the bpp is out of control. Can I arbitrarily choose the bpp?

**Comment by YodaEmbedding** [comment]
Depends on the image contents and image resolution.

The RD curves uses the average over 24 images of size 768x512 from the Kodak dataset.

**Comment by BeamReflection** [comment]
> Depends on the image contents and image resolution.
> 
> A single point on the RD curve represents a single model at a particular quality. The point is generated by taking the average over 24 images of size 768x512 from the Kodak dataset.

Thanks for your answer.
I have another question. For a certain point on the RD curve, my bpp value is much larger at the same PSNR value. I hardly changed the parameters of the model, just changed lambda, and I used the CLIC training set.
Maybe I didn't have enough epochs? I selected 13,000 images from CLIC and trained for 500 epochs.

**Comment by lyysl** [comment]
I have some similar issues, the image quality ranges are different for traditional codecs and neural network, I want to ask you just take some evenly distributed value between the ranges like 10,20,...,100 for traditional codecs or you first give an certain metric values and use python -m compressai.utils.find_close ?
Thx
